camino-messenger-bot.yaml

```diff
@@ -0,0 +1,38 @@
+# Example config
```

Comment:
is this intended? I think we already have example configs?

Reply:
Will remove it